pylutron_caseta/color_value.py

```diff
@@ -0,0 +1,192 @@
+# protocol which defines a method that returns a dictionary of the leap command given a zone string
```

Comment:
Is this supposed to be a module doc string?

Reply:
Its an old comment im going to remove and make sure everything is commented as needed